Example/Tests/ScanHistoryViewModelTests.swift

```diff
@@ -107,6 +107,24 @@ class ScanHistoryViewModelTests: XCTestCase {
         XCTAssertTrue(reloadedSectionHeader)
     }
 
+    func testPendingScan() {
+        setUpVM(scans: [], isScanningSessionExpirable: true)
+
+        var rowReloadedIndexPath: IndexPath?
+        viewModel.reloadRowObserver = { indexPath in
+            rowReloadedIndexPath = indexPath
+        }
+
+        var reloadedSectionHeader = false
+        viewModel.reloadSectionHeaderObserver = { _ in
+            reloadedSectionHeader = true
+        }
+
+        viewModel.didScan(barcode: "0000000001", with: .pending)
+        XCTAssertNil(rowReloadedIndexPath)
+        XCTAssertFalse(reloadedSectionHeader)
```

Comment:
So the coordinator can't be initialized because it requires the camera, so any tests auto-fail on the simulator, which I guess is why there were no coordinator tests already

Reply:
Got it! Out of scope for this PR, but I wonder if we could just fall back to a black screen on a simulator instead of failing.